docs/developer/users.mdx

```diff
@@ -9,6 +9,12 @@ import Chart from "@site/components/Chart";
 
 This guide describes how to use the Saleor GraphQL API to register a new account as a customer and do common account management operations.
 
+## Multiple channels and users
+
+Creating a new account requires specifying a channel. It is needed because you can configure different email templates on a by-channel basis.
```

Comment:
```suggestion
Creating a new account requires specifying a channel. It is needed because you can configure different email templates on a per-channel basis.
```